code-of-conduct.md

```diff
@@ -1 +1,76 @@
 # Triangle DevOps Code of Conduct
+
+Triangle DevOps is dedicated to providing a harassment-free community experience
+for everyone, regardless of gender, gender identity and expression, sexual
+orientation, disability, physical appearance, body size, race, age or religion.
+We do not tolerate harassment of community participants in any form.
+
+Community participants violating these rules may be sanctioned or expelled from
```

Comment:
+1 @mmzyk 

Both that organizers will be _protected by_ and _held to_ the CoC.